### ISSUE TITLE
Reduce count of staffcp form pages

### ISSUE THREAD

**Issue by Aberdeener** [NamelessMC/Nameless#1973]
Currently in most staffcp sections (for example Groups), where is a tpl file to make a new group, and a tpl file to edit an existing group (`groups_new.tpl`, `groups_edit.tpl` respectively). 

This seems redundant as both are the exact same file.

When I made the Announcements system I made only one tpl file (`announcements_form.tpl`). 
The backend knows whether to edit or create a new announcement from a simple hidden input.

By converting all other forms to this single tpl file, we are doing an important thing: reducing workload for developers, as they (we) wont have to edit two files.

Seems like something @Derkades would be interested in helping with :p 

What do you think?

**Comment by CqldFire** [comment]
I completely agree with this, having multiple files being pretty much exactly the same created a lot of extra work for me when making the new default panel template.

**Comment by partydragen** [comment]
well i like the idea but then we need to create values for everything then we can anyways add restore ability if validation fails that not everything have
